src/main/service/listManipulation/CrimeListsService.ts

```diff
@@ -91,18 +92,16 @@ export class CrimeListsService {
         let linkedCases = '';
         let listingNotes = '';
 
-        if (hearing?.case) {
-            hearing.case.forEach(cases => {
-                linkedCases = '';
-                if (cases?.caseLinked) {
-                    cases.caseLinked.forEach(caseLinked => {
-                        linkedCases += caseLinked.caseId.trim();
-                        linkedCases += helperService.stringDelimiter(linkedCases?.length, ',');
-                    });
-                }
-                cases['linkedCases'] = linkedCases?.replace(/,\s*$/, '').trim();
-            });
-        }
+        hearing?.case.forEach(cases => {
+            linkedCases = '';
+            if (cases?.caseLinked) {
```

Comment:
The question mark can be removed to because we are inside the forEach loop of the case element

Reply:
Question mark removed